src/components/OrganizationListTablePanel.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { OrganizationBundle } from '@pdc/sdk';
+import { Panel, PanelBody } from './Panel';
+import { OrganizationListTable } from './OrganizationListTable';
+
+interface OrganizationListTablePanelProps {
+	organizations?: OrganizationBundle;
+	loading?: boolean;
+}
+
+const DEFAULT_COLUMNS = ['Organization Name', 'Organization EIN'];
+
+export const OrganizationListTablePanel = ({
+	organizations = undefined,
+	loading = false,
+}: OrganizationListTablePanelProps) => {
+	const generateFallbackMessage = () =>
+		loading ? 'Loading…' : 'No data available.';
+
+	return (
+		<Panel>
+			<PanelBody
+				padded={organizations !== undefined && organizations.entries.length > 0}
```

Comment:
Could you put this in a named utility method, that way it's clear from reading what it is checking for and why (this will also mean you can remove that portion of the commit message!)

Maybe something like `hasOrganizations()`